### PR TITLE
Fix Building and Verifying PASTs example in documentation

### PR DESCRIPTION
**Pull request by Edwin-Luijten**
Move setClaims() above setExpiration() in Building and Verifying PASTs example.
If using setClaims() after a method that sets a claim, it will overwrite previous set claims.

### CODE REVIEW

File: docs/02-PHP-Library/README.md [hunk 1]
```diff
@@ -52,15 +52,14 @@ $token = (new JsonToken())
     ->setKey($sharedKey)
     ->setVersion(Version2::HEADER)
     ->setPurpose('local')
+    // Store arbitrary data
+    ->setClaim('security', 'Hello world')
```
Comment (jdreesen) [inline docs/02-PHP-Library/README.md]: There's something wrong here...

File: docs/02-PHP-Library/README.md [hunk 2]
```diff
@@ -52,15 +52,14 @@ $token = (new JsonToken())
...
+        'security' => 'Now as easy as PIE'
```
Comment (jdreesen) [inline docs/02-PHP-Library/README.md]: There's something wrong here...

File: docs/02-PHP-Library/README.md [hunk 3]
```diff
@@ -52,15 +52,15 @@ $token = (new JsonToken())
     ->setKey($sharedKey)
     ->setVersion(Version2::HEADER)
     ->setPurpose('local')
-    // Set it to expire in one day
-    ->setExpiration(
-        (new DateTime())->add(new DateInterval('P01D'))
-    )
     // Store arbitrary data
-    ->setClaims([
+    ->setClaim([
```
Comment (jdreesen) [inline docs/02-PHP-Library/README.md]: s/setClaim/setClaim**s**/

**Comment by jdreesen** [comment]
> If using setClaims() after a method that sets a claim, it will overwrite previous set claims.

I don't think this is true anymore with f3f503657bbb48b2268c037f9ffec5485edd91c7.

**Comment by Edwin-Luijten** [comment]
Your right =). It's fixed on master. Then of course this pull can be closed/removed.

**Pull request closed without merging**